x11-packages/rofi/build.sh

```diff
@@ -1,8 +1,9 @@
 TERMUX_PKG_HOMEPAGE=https://github.com/DaveDavenport/rofi
 TERMUX_PKG_DESCRIPTION="A window switcher, application launcher and dmenu replacement"
-TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_LICENSE="MIT, X11"
```

Comment:
Seems like MIT describes rofi's license better, the COPYING file doesn't have the extra paragraph that distinguishes MIT from X11 license, so I think this change should be dropped